app/js/components/manageApps/ActionAddonModal.jsx

```diff
@@ -51,19 +51,20 @@ export default class DeleteModal extends Component {
               className="btn btn-danger"
               data-dismiss="modal"
               id="cancel-btn"
-              onClick={() => handleDelete(app.name, appUuid)}
-            >Delete</button>
+              onClick={() => (action === 'delete')? handleAction(appUuid, app.name): handleAction(appUuid, action)}
+            >{(action === 'delete')? "Delete" : "Stop"}</button>
```

Comment:
You should be more consistent with the quotes. I think you want to use `'` instead of `"` for `"Delete"`